FTHEL8/Database/Database.cs

```diff
@@ -0,0 +1,432 @@
+﻿using FTHEL8.Models;
+using Newtonsoft.Json;
+using System.Text;
+
+namespace FTHEL8.Data
+{
+    public class Database
```

Comment:
Ez nem egy adatbázis, inkább egy adatelérést megvalósító komponens, amit én inkább a félrevezetés elkerülése végett DataAcces-nek vagy DataHandler-nek neveztem volna el. Mivel minden metódusa Static, ezért nehéz lesz tesztelni, mockolni a metódusait. Inkább egy példányosítható osztálynak kellett volna lennie. A static class egyfajta sub namespace metódusoknak.